packages/core/test/tools/mocky.js

```diff
@@ -51,10 +51,10 @@ const mockRpcGetPresignedPostCall = (key) => {
     });
 };
 
-const mockRpcFail = (error) => {
+const mockRpcFail = (error, status = 400) => {
```

Comment:
HTTP 400 is the default error code for manual failures in the RPC API now, so I've set it as the default for this test